Type validator is broken on python3.9.0-1

<!--
Thanks for helping us improve Prisma Client Python! 🙏 Please follow the sections in the template and provide as much information as possible about your problem, e.g. by enabling additional logging output.

See https://prisma-client-py.readthedocs.io/en/latest/logging/ for how to enable additional logging output.
-->

## Bug description

<!-- A clear and concise description of what the bug is. -->

`tests/test_validator.py::test_valid` and `tests/test_validator.py::test_recursive` fail on `python3.9.1`

`test_valid` output:

```
―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― test_valid ――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――

    def test_valid() -> None:
        """Basic usage with correct data"""
>       validated = validate(types.IntFilter, {'equals': 1})

tests/test_validator.py:14: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/private/tmp/tox/prisma-client-py/py39/lib/python3.9/site-packages/prisma/validator.py:81: in validate
    instance = model.parse_obj(data)
pydantic/main.py:578: in pydantic.main.BaseModel.parse_obj
    ???
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   pydantic.error_wrappers.ValidationError: 7 validation errors for IntFilter
E   not_in
E     field required (type=value_error.missing)
E   lt
E     field required (type=value_error.missing)
E   lte
E     field required (type=value_error.missing)
E   gt
E     field required (type=value_error.missing)
E   gte
E     field required (type=value_error.missing)
E   in
E     field required (type=value_error.missing)
E   not
E     field required (type=value_error.missing)

pydantic/main.py:406: ValidationError
```

## How to reproduce

<!--
Steps to reproduce the behavior:
1. Go to '...'
2. Change '....'
3. Run '....'
4. See error
-->

`tox -e py39 -- tests/test_validator.py`

## Expected behavior

<!-- A clear and concise description of what you expected to happen. -->

Tests should pass

## Environment & setup

<!-- In which environment does the problem occur -->

- OS: <!--[e.g. Mac OS, Windows, Debian, CentOS, ...]--> Mac OS
- Database: <!--[PostgreSQL, MySQL, MariaDB or SQLite]--> SQLite
- Python version: <!--[Run `python -V` to see your Python version]--> Python 3.9.1

--- a/requirements/base.txt
+++ b/requirements/base.txt
@@ -4,4 +4,4 @@ pydantic>=1.8.0
 click>=7.1.2
 python-dotenv>=0.12.0
 contextvars; python_version < '3.7'
-typing-extensions>=3.7; python_version < '3.9'
+typing-extensions>=3.7; python_version < '3.9.2'

--- a/requirements/base.txt
+++ b/requirements/base.txt
@@ -3,5 +3,5 @@ jinja2>=2.11.2
 pydantic>=1.8.0
 click>=7.1.2
 python-dotenv>=0.12.0
+typing-extensions>=3.7
 contextvars; python_version < '3.7'
-typing-extensions>=3.7; python_version < '3.9.2'

--- a/src/prisma/_types.py
+++ b/src/prisma/_types.py
@@ -1,23 +1,12 @@
-import sys
 from typing import Callable, Coroutine, TypeVar, Any
+from typing_extensions import (
+    TypedDict as TypedDict,
+    Protocol as Protocol,
+    Literal as Literal,
+    runtime_checkable as runtime_checkable,
+)
 
 from pydantic import BaseModel
-
-
-if sys.version_info >= (3, 9, 2):
-    from typing import (  # pylint: disable=no-name-in-module, unused-import
-        TypedDict as TypedDict,
-        Protocol as Protocol,
-        Literal as Literal,
-        runtime_checkable as runtime_checkable,
-    )
-else:
-    from typing_extensions import (
-        TypedDict as TypedDict,
-        Protocol as Protocol,
-        Literal as Literal,
-        runtime_checkable as runtime_checkable,
-    )
 
 
 Method = Literal['GET', 'POST']

--- a/src/prisma/_types.py
+++ b/src/prisma/_types.py
@@ -4,7 +4,7 @@ from typing import Callable, Coroutine, TypeVar, Any
 from pydantic import BaseModel
 
 
-if sys.version_info >= (3, 9):
+if sys.version_info >= (3, 9, 2):
     from typing import (  # pylint: disable=no-name-in-module, unused-import
         TypedDict as TypedDict,
         Protocol as Protocol,

--- a/src/prisma/generator/templates/_header.py.jinja
+++ b/src/prisma/generator/templates/_header.py.jinja
@@ -25,10 +25,6 @@ from typing import (
     overload,
     cast,
 )
-
-if sys.version_info >= (3, 9, 2):
-    from typing import TypedDict, Literal
-else:
-    from typing_extensions import TypedDict, Literal
+from typing_extensions import TypedDict, Literal
 
 

--- a/src/prisma/generator/templates/_header.py.jinja
+++ b/src/prisma/generator/templates/_header.py.jinja
@@ -26,7 +26,7 @@ from typing import (
     cast,
 )
 
-if sys.version_info >= (3, 9):
+if sys.version_info >= (3, 9, 2):
     from typing import TypedDict, Literal
 else:
     from typing_extensions import TypedDict, Literal

